src/components/Toggle/__stories__/Template.stories.tsx

```diff
@@ -0,0 +1,6 @@
+import { ComponentStory } from '@storybook/react'
+import Toggle from '..'
+
+export const Template: ComponentStory<typeof Toggle> = args => (
+  <Toggle {...args} />
+)
```

Comment:
There is an issue with Toggle, the component doesn't seems to be working any more on stories:


https://user-images.githubusercontent.com/15812968/205659247-78fe1c89-5bfa-4ee5-9456-9791d7686cc6.mov

Plus the onChange is messed up in code snippet:
![Screenshot 2022-12-05 at 15 18 57](https://user-images.githubusercontent.com/15812968/205659335-855b7326-ac03-46e7-8987-588f3864996b.png)

Reply:
should be fixed